ansible-ipi-install/README.md

```diff
@@ -1,988 +1,3 @@
-| `extramanifests` | include any extra manifests files                                                                                                                                |
-| `install`        | Deploy OpenShift                                                                                                                                                 |
-| `cleanup`        | clean up the environment within the provisioning node. Does not remove networking                                                                                |
-
-## How to use the Ansible tags
-
-The following is an example on how to use the `--tags` option. In this example, we will just install the packages to the provision node.
-
-Example 1
-
-```sh
-ansible-playbook -i inventory/hosts playbook.yml --tags "validation,packages"
-```
-
-In the next example, we will show how to call multiple tags at the same time.
-
-Example 2
-
-```sh
-ansible-playbook -i inventory/hosts playbook.yml --tags "validation,network,packages"
-```
-
-The example above calls for the setup of the networking and installation of the packages from the Ansible playbook. Only the tasks with these specific tags will run.
-
-NOTE: Due to the dependencies in the playbook, the `validation` tag is always required. 
-
-## Skipping particular tasks using Ansible tags
-
-In the event that you want to always skip certain tasks of the playbook this can be done via the `--skip-tag` option.
-
-We will use similar example as above where we want to skip the network setup and the package installation.
-
-Example 1
-
-```sh
-ansible-playbook -i inventory/hosts playbook.yml --skip-tags "network,packages"
-```
-
-# Appendix B. Using a proxy with your Ansible playbook
-
-When running behind a proxy, it is important to properly set the environment
-to handle such scenario such that you can run the Ansible playbook. In order
-to use a proxy for the ansible playbook set the appropriate variables within
-your `inventory/hosts` file. These values will also be included within your
-generated `install-config.yaml` file.
-
-```sh
-# (Optional) Provide HTTP proxy settings
-#http_proxy=http://USERNAME:PASSWORD@proxy.example.com:8080
-
-# (Optional) Provide HTTPS proxy settings
-#https_proxy=https://USERNAME:PASSWORD@proxy.example.com:8080
-
-# (Optional) comma-separated list of hosts, IP Addresses, or IP ranges in CIDR format
-# excluded from proxying
-# NOTE: OpenShift does not accept '*' as a wildcard attached to a domain suffix
-# i.e. *.example.com
-# Use '.' as the wildcard for a domain suffix as shown in the example below.
-# i.e. .example.com
-#no_proxy_list="172.22.0.0/24,.example.com"
-```
+Please check <https://openshift-kni.github.io/baremetal-deploy/> for the `Ansible-Playbook-Install` to get access to the information that used to live here.
```

Comment:
For the latest information on how to install via the Ansible playbook, check out the `Ansible-Playbook-Install` document at <https://openshift-kni.github.io/baremetal-deploy/>